parent/pom.xml

```diff
@@ -622,6 +631,11 @@
       <name>SpringSource Enterprise Bundle Repository - External Bundle Releases</name>
       <url>http://repository.springsource.com/maven/bundles/external</url>
     </repository>
+
+    <repository>
+      <id>fix-maven-snapshots-issue-with-nexus</id>
+      <url>allow-parent-pom-resolving-when-snapshot</url>
+    </repository>
```

Comment:
Do you know why this wasn't necessary when the BOM was the parent of this?
Any specific errors if we remove this that you already encountered? (Haven't tested yet myself)

Reply:
I have to check that one again. This was a bit of an attempt to keep the effective pom the same after the refactoring, but I'm not sure it is really needed.

Reply:
I assume the `repository` entry must be present in every pom (https://github.com/camunda/camunda-bpm-platform/commit/4487ed59c30b75f68c607042a40cae083baa2efd has added this, but I don't understand the reasoning behind it).

Previously we had the BOM as the parent of every pom in the repo except for camunda-root, and that is where this dummy repo was configured. With the changes we made, it is probably sufficient to have them in camunda-root only. I will try that.